src/_locales/zh_TW/messages.json

```diff
@@ -0,0 +1,496 @@
+  },
+  "loading": {
+    "message": "載入中...",
+    "description": "A general loading message.",
+    "hash": "e2a5d4627618472a68f4cb8895e93f30"
+  },
+  "couldNotGenerateQrCode": {
+    "message": "無法產生二維條碼",
+    "hash": "87e71800b895b0e9695da467734df0e5"
+  },
+  "couldNotReceiveActiveTab": {
+    "message": "無法啟動執行頁籤",
+    "hash": "2b4c5f788e39f7ae35dcca2f96e1e5d4"
+  },
+  "errorDownloadingFile": {
+    "message": "無法下載檔案",
+    "description": "Error shown when a file could not be downloaded. Do not mention a file type or name, but just keep it general.",
+    "hash": "8f5756df0d5dd4d25f01ca4b4ff0ec44"
+  },
+  "errorPermissionRequired": {
+    "message": "需要權限才能繼續",
+    "description": "Error shown when a permission is declined and the thing the user triggered cannot be executed.",
+    "hash": "9742f751499f8c2acf32d83df83c36b9"
+  },
+  "errorPermissionRequestFailed": {
+    "message": "權限請求失敗",
+    "description": "Error shown when a permission request failed. This indicates a technical error and does NOT mean the user declined the permission.",
+    "hash": "54aaf50f8bdf6154677a4ff7fd384310"
+  },
+  "errorQrCodeOverflow": {
+    "message": "無法產生二維條碼：內容太長",
+    "description": "Error shown when QR code cannot be generated due to overflow in input size.",
+    "hash": "36c48d76203a1f1362b32abf72af5d8b"
+  },
+  "requestDownloadPermissionForQr": {
+    "message": "需要下載權限才能保存二維條碼",
+    "description": "Shown, when the user is asked to allow the download permission to save the QR code.",
+    "hash": "6e5a5fe9a077400ba6e4dc66a6dfd44a"
+  },
+  "lowContrastRatioInfo": {
+    "message": "對比度較低，可能在某些讀取設備上無法讀取。",
+    "hash": "90432f33baa13153accd007ae2e23e99"
+  },
+  "lowContrastRatioWarning": {
+    "message": "對比度太低，可能無法在所有讀取裝置上識別。",
+    "hash": "f4264a9b8b24922e4dff589f611b5e2f"
+  },
+  "lowContrastRatioError": {
+    "message": "二維條碼顏色與背景色太相近！",
+    "description": "The message shown when the color for the QR code and the background are equal",
+    "hash": "a2d1d2fdc32ecb73ffdf0ab537e46f32"
+  },
+  "messageAutoSelectColorButton": {
+    "message": "使用對比度高的顏色",
+    "description": "The text of a button that sets a new color with a sufficient contrast.",
+    "hash": "9d422df9f61f2b45e272ae3451ca3ded"
+  },
+
+  // tips
+  "tipYouLikeAddon": {
+    "message": "喜歡這個附加元件",
+    "description": "A tip shown to remind the user to rate the addon.",
+    "hash": "0f5680a5a42bcebf793a44b9dfcf79a1"
+  },
+  "tipYouLikeAddonButton": {
+    "message": "請給評價",
+    "description": "Button for the tip shown to remind the user to rate the addon.",
+    "hash": "821d7f942b02e2b56cd0424d3ab65de8"
+  },
+  "tipSaveQrCode": {
+    "message": "您知道可以保存二維條碼嗎？",
+    "description": "A tip shown to get the user to find the save option.",
+    "hash": "84f0a0027a58c98d9158ab9894fe8427"
+  },
+  "tipSaveQrCodeLink": {
+    "message": "https://github.com/rugk/offline-qr-code/wiki/FAQ#how-to-save-the-qr-code-on-disk",
```

Comment:
Well, as you seem to have that translated in the wiki now, you can change the link to the new wiki page.

(Please do this for all links.)

Ref:
> Do not forget that files in the project's main repository itself often refer to the wiki, so remember to also replace the links in here.

https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#translation-of-wiki

Reply:
Edited.